portality/view/api_v1.py

```diff
@@ -256,6 +256,7 @@ def retrieve_journal(journal_id):
 
 @blueprint.route("/bulk/applications", methods=["POST"])
 @api_key_required
+@swag(swag_summary='Create applications in bulk', swag_spec=ApplicationsBulkApi.create_swag())  # must be applied after @api_key_(optional|required) decorators. They don't preserve func attributes.
```

Comment:
we usually add ``` <span class="red">[Authenticated, not public]</span>``` at the end of the route description just to make people aware of what's public at a glance